database/influxdb/influxdb_database.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/python
+
+# (c) 2016, Kamil Szczygiel <kamil.szczygiel () intel.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: influxdb_database
+short_description: Manage InfluxDB databases
+description:
+    - Manage InfluxDB databases
+version_added: 2.1
+author: "Kamil Szczygiel (@kamsz)"
+requirements:
+    - "python >= 2.6"
+    - "influxdb >= 0.9"
+options:
+    hostname:
+        description:
+            - The hostname or IP address on which InfluxDB server is listening
+        required: True
+    username:
+        description:
+            - Username that will be used to authenticate against InfluxDB server
+        default: root
+    password:
+        description:
+            - Password that will be used to authenticate against InfluxDB server
+        default: root
+    port:
+        description:
+            - The port on which InfluxDB server is listening
+        default: 8086
+    database_name:
+        description:
+            - Name of the database that will be created/destroyed
+        required: True
+    state:
+        description:
+            - Determines if the database should be created or destroyed
+        choices: ['present', 'absent']
+        default: present
+'''
+
+EXAMPLES = '''
+# Example influxdb_database command from Ansible Playbooks
+- name: Create database
+    influxdb_database:
+      hostname: "{{influxdb_ip_address}}"
+      database_name: "{{influxdb_database_name}}"
+      state: present
+
+- name: Destroy database
+    influxdb_database:
+      hostname: "{{influxdb_ip_address}}"
+      database_name: "{{influxdb_database_name}}"
+      state: absent
+
+- name: Create database using custom credentials
+    influxdb_database:
+      hostname: "{{influxdb_ip_address}}"
+      username: "{{influxdb_username}}"
+      password: "{{influxdb_password}}"
+      database_name: "{{influxdb_database_name}}"
+      state: present
+'''
+
+RETURN = '''
+#
```

Comment:
Could you please add this in the RETURN?

```#only defaults```

We need to work on doing this better so these defaults get inserted when the docs are generated. That placeholder will help once that happens.